hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/FeeManager.java

```diff
@@ -179,9 +179,6 @@ public FeeCalculator createFeeCalculator(
         // If it is not known, that is, if we have no fee data for that transaction, then we MUST NOT execute that
         // transaction! We will not be able to charge appropriately for it.
         final var feeData = getFeeData(functionality, consensusTime, subType);
-        if (feeData == null) {
```

Comment:
Would it be better to check against DEFAULT_FEE_DATA, rather than remove this check entirely?

Reply:
the getFeeData will return the DEFAULT_FEE_DATA thats why I don't need it here. it  cannot be null

Reply:
This is a leftover from a fix we did a while ago. If no fee data is available, we must not fail but use the default instead. (One could question if this is the desired behavior, but it is what mono does.)